docs/ruleset_functions/get_loop_zone_list_w_area.md

```diff
@@ -0,0 +1,37 @@
+
+## get_loop_zone_list_w_area
```

Comment:
Please change the name of function to get_HW_loop_zone_list_w_area.

Reply:
Updated.